build.sbt

```diff
@@ -41,6 +41,7 @@ def test(scope: String = "test"): Seq[ModuleID] = Seq(
   "org.scalatestplus.play" %% "scalatestplus-play" % scalaTestPlusVersion % scope,
   "org.scalamock" %% "scalamock" % scalaMockVersion % scope,
   "org.pegdown" % "pegdown" % pegdownVersion % scope,
+  "uk.gov.hmrc" %% "crypto-json-play-28" % "7.6.0",
```

Comment:
do we have something to support Play 3.0? as we migrated to higher version as the moment